starknet_devnet/blueprints/rpc/routes.py

```diff
@@ -128,6 +128,11 @@ def parse_body(body: dict) -> Tuple[Callable, Union[List, dict], int]:
         )
 
     if not isinstance(params, (List, Dict)):
-        raise RpcError(code=RpcErrorCode.INVALID_PARAMS.value, message="Invalid params")
+        raise RpcError(
+            code=RpcErrorCode.INVALID_PARAMS.value,
+            # fmt: off
+            message="Invalid \"params\" type. Value of \"params\" must be a dict or list",
+            # fmt: on
```

Comment:
For whatever reason black doesn't like escaped double quotes, so it converts the message to `'Invalid "params" type...'` but then pylint complains that invalid quotes (`'` instead of `"`) are being used.

So it's either disabling autoformatting or pylint here